<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="TejaswiniArun" failures="1" tests="1" name="Error Validations" time="0.159" errors="0" timestamp="2024-08-05T17:36:23 SAST">
  <testcase classname="TejaswiniFrameworkcompany.ErrorValidations" name="ErrorMessage" time="0.032">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at TejaswiniFrameworkcompany.ErrorValidations.ErrorMessage(ErrorValidations.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- ErrorMessage -->
  <testcase classname="TejaswiniFrameworkcompany.ErrorValidations" name="submitOrder" time="0.0">
    <ignored/>
  </testcase> <!-- submitOrder -->
</testsuite> <!-- Error Validations -->
